tools/livecd-creator

```diff
@@ -50,6 +50,9 @@ def parse_options(args):
                       help="Title used by syslinux.cfg file"),
     imgopt.add_option("", "--product", type="string", dest="product",
                       help="Product name used in syslinux.cfg boot stanzas and countdown"),
+    imgopt.add_option("", "--nomac", action="store_true",
```

Comment:
Can we call this `--nomacboot`?

Reply:
Option --nomac renamed to --nomacboot as requested